packages/web3/src/providers.exports.ts

```diff
@@ -18,5 +18,4 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 export { Eip1193Provider, SocketProvider } from 'web3-utils';
 
 export * as http from 'web3-providers-http';
-export * as ipc from 'web3-providers-ipc';
```

Comment:
There are tests failing and we will also need to mention this in migration guide. @mpetrunic Many Thanks

Reply:
It's a bit more severe than that, need to do conditional module loading in web3-core requestmanager 😞